Documentation/project-docs/developer-guide.md

```diff
@@ -178,34 +178,32 @@ Under the src directory is a set of directories, each of which represents a part
 
 For example the src\System.Diagnostics.DiagnosticSource directory holds the source code for the System.Diagnostics.DiagnosticSource.dll assembly.
 
-You can build the DLL for System.Diagnostics.DiagnosticSource.dll by going to the `src\System.Diagnostics.DiagnosticsSource\src` directory and typing `msbuild`. The DLL ends up in `bin\AnyOS.AnyCPU.Debug\System.Diagnostics.DiagnosticSource` as well as `bin\runtime\[BuildConfiguration]`.
+You can build the DLL for System.Diagnostics.DiagnosticSource.dll by going to the `src\System.Diagnostics.DiagnosticsSource\src` directory and typing `dotnet msbuild`. The DLL ends up in `bin\AnyOS.AnyCPU.Debug\System.Diagnostics.DiagnosticSource` as well as `bin\runtime\[BuildConfiguration]`.
 
 You can build the tests for System.Diagnostics.DiagnosticSource.dll by going to
-`src\System.Diagnostics.DiagnosticSource\tests` and typing `msbuild`.
+`src\System.Diagnostics.DiagnosticSource\tests` and typing `dotnet msbuild`.
 
-Some libraries might also have a ref and/or a pkg directory and you can build them in a similar way by typing `msbuild` in that directory.
+Some libraries might also have a ref and/or a pkg directory and you can build them in a similar way by typing `dotnet msbuild` in that directory.
 
 For libraries that have multiple build configurations the configurations will be listed in the `<BuildConfigurations>` property group, commonly found in a configurations.props file next to the csproj. When building the csproj for a configuration the most compatible one in the list will be choosen and set for the build. For more information about `BuildConfigurations` see [project-guidelines](../coding-guidelines/project-guidelines.md).
 
 **Examples**
 
 - Build project for Linux for netcoreapp
 ```
-msbuild System.Net.NetworkInformation.csproj /p:OSGroup=Linux
+dotnet msbuild System.Net.NetworkInformation.csproj /p:OSGroup=Linux
 ```
 
 - Build project for uap (not if trying to build on non-windows you also need to specify OSGroup=Windows_NT)
 ```
-msbuild System.Net.NetworkInformation.csproj /p:TargetGroup=uap
+dotnet msbuild System.Net.NetworkInformation.csproj /p:TargetGroup=uap
 ```
 
 - Build release version of library
 ```
-msbuild System.Net.NetworkInformation.csproj /p:ConfigurationGroup=Release
+dotnet msbuild System.Net.NetworkInformation.csproj /p:ConfigurationGroup=Release
 ```
 
-**Note:** If building in a non-Windows environment, call `<repo-root>/Tools/msbuild.sh` instead of just `msbuild`.
```

Comment:
I think this might be a good place to mention `msbuild` is a placeholder for `dotnet msbuild` or `msbuild` depending on whether `dotnet` or `msbuild` is on your path.